src/test/scala/io/iohk/ethereum/blockchain/sync/SyncControllerSpec.scala

```diff
@@ -958,18 +965,22 @@ class SyncControllerSpec extends FlatSpec with Matchers {
 
     peerMessageBus.ignoreNoMsg()
 
-    syncController ! SyncController.StartSync
+    syncController ! SyncController.Start
+
+    syncController.getSingleChild("regular-sync") ! HandshakedPeers(Map(
+      peer -> PeerInfo(peer1Status, forkAccepted = true, totalDifficulty = peer1Status.totalDifficulty, maxBlockNumber = 0)))
 
     //Send block headers request
     etcPeerManager.expectMsg(EtcPeerManagerActor.SendMessage(
       GetBlockHeaders(Left(expectedMaxBlock + 1), syncConfig.blockHeadersPerRequest, 0, reverse = false),
       peer.id))
     peerMessageBus.expectMsg(Subscribe(PeerDisconnectedClassifier(PeerSelector.WithId(peer.id))))
+    peerMessageBus.expectMsg(Subscribe(PeerDisconnectedClassifier(PeerSelector.WithId(peer.id)))) // TODO: why twice?
```

Comment:
Should we address this `TODO` or maybe create a task for that? This could be related with the `BlacklistSupport` comment as it seems to apply as well to the `PeerListSupport` trait.